airflow/config_templates/default_airflow.cfg

```diff
@@ -511,6 +511,10 @@ scheduler_zombie_task_threshold = 300
 # DAG definition (catchup)
 catchup_by_default = True
 
+# Should DAGs be schedule to execute at the 'start' or 'end'
+# of their schedule interval.
+execute_at_interval_end = True
```

Comment:
I wonder if `schedule_at_interval_end` is a better/clearer name.

I'm probably splitting hairs

Reply:
I'm all for hair splitting if it means clearer usage down the road, and is more consistent with established nomenclature.

Part of my confusion is that `schedule` seams to have multiple meanings: A job scheduled daily will be scheduled to run once the day has finished. So there's the declared schedule, and then the implied schedule (execution time).

I have no vested interest in the name, just let me know if you'd like to change it.

Either way, maybe it would be worth adding (in a separate PR) a dictionary of terms and precise definitions? I'd be happy submit that after this.

Reply:
All these terms are confusing and overlapping.

The reason I wanted to avoid `execute` is because the `execution_date` property of the task  will remain constant non matter what this setting. So I think this does need to change, but I'm not at all convinced my wording is better.

> Either way, maybe it would be worth adding (in a separate PR) a dictionary of terms and precise definitions? I'd be happy submit that after this.

Yes, that would be great!

Reply:
I've updated the name, I like it, and it makes sense. Also made a few more very small documentation changes, and squashed all the commits. Please let me know if you'd like to change anything else. 

Do you think there's any way this could get cherry picked for the next stable release? Right now I'm using a hacked up local version to get this behaviour, and it would be great to get back to upstream stable.